libs/application/templates/car-recycling/src/fields/VehiclesOverview/VehiclesOverview.tsx

```diff
@@ -285,9 +285,17 @@ const VehiclesOverview: FC<FieldBaseProps> = ({
         <Label>{formatMessage(carRecyclingMessages.cars.overview)}</Label>
       </Box>
 
-      {currentVehiclesList
-        .slice(ITEMS_ON_PAGE * (page - 1), ITEMS_ON_PAGE * page)
-        .map((vehicle: VehicleDto, index) => {
+      {loadingCurrent ? (
+        <SkeletonLoader
+          repeat={5}
+          space={2}
+          height={130}
+          borderRadius="large"
+        />
+      ) : currentVehiclesList && currentVehiclesList.length > 0 ? (
```

Comment:
currentVehiclesList is always an array. This extra check for if it's truthy is not needed
```suggestion
      ) : currentVehiclesList.length > 0 ? (
```

Reply:
fixed